websites/V/VanMoof/presence.ts

```diff
@@ -50,28 +50,29 @@ presence.on("UpdateData", async function () {
 				const hash = window.location.hash.replace("#", "");
 
 				if (urlpath[3].startsWith("configure-"))
-					return (presenceData.state = "Configuring");
-
```

Comment:
you should simply remove the return to the next line and add brackets

Reply:
wdym?

Reply:
I mean you should move the return to the line below the presenceData.state assignment and not add the else below